app/models/RecommendationsRepository.scala

```diff
@@ -0,0 +1,216 @@
+package models
+
+import javax.inject.Inject
+
+import models.RecommendationsJsonFormats._
+import play.api.libs.json.Json
+import play.modules.reactivemongo.ReactiveMongoApi
+import reactivemongo.api.Cursor.FailOnError
+import reactivemongo.api.{QueryOpts, ReadPreference}
+import reactivemongo.api.commands.{UpdateWriteResult, WriteResult}
+import reactivemongo.bson.{BSONDateTime, BSONDocument, BSONObjectID}
+import reactivemongo.play.json.collection.JSONCollection
+import utilities.DateTimeUtility
+
+import scala.annotation.switch
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{ExecutionContext, Future}
+
+// this is not an unused import contrary to what intellij suggests, do not optimize
+import reactivemongo.play.json.BSONFormats.BSONObjectIDFormat
+import reactivemongo.play.json.BSONFormats.BSONDateTimeFormat
+
+case class RecommendationInfo(email: Option[String],
+                              name: String,
+                              topic: String,
+                              recommendation: String,
+                              submissionDate: BSONDateTime,
+                              updateDate: BSONDateTime,
+                              approved: Boolean = false,
+                              decline: Boolean = false,
+                              pending: Boolean = true,
+                              done: Boolean = false,
```

Comment:
Make sure done field is updated automatically once an Admin schedules a session.